L1Trigger/VertexFinder/python/l1tVertexProducer_cfi.py

```diff
@@ -38,7 +38,7 @@
         # Track word limits (256 binns): [-20.46912512, 20.46912512, 0.15991504]
         FH_HistogramParameters = cms.vdouble(-20.46912512, 20.46912512, 0.15991504),
         # The number of vertixes to return (i.e. N windows with the highest combined pT)
-        FH_NVtx = cms.uint32(10),
+        FH_NVtx = cms.uint32(1),
```

Comment:
Just to note that this should really be 1 in order to follow what the FW will do.